promql/lex.go

```diff
@@ -127,6 +127,7 @@ const (
 	itemDIV
 	itemLAND
 	itemLOR
+	itemLUnless
```

Comment:
Hmm, this probably should be itemUNLESS for consistency with the other operators.